src/main/java/utils/StringToDate.java

```diff
@@ -0,0 +1,141 @@
+package utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * This class returns literal dates converted to date format.
+ */
+public class StringToDate {
+
+    SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
+
+    /**
+     * Converts literal date String to a date format.
+     *
+     * @param stringToConvert a literal date String.
+     * @return a date format.
+     */
+    public Date convert(final String stringToConvert) {
+        if (stringToConvert == null) {
+            throw new IllegalArgumentException("Value is null");
+        }
+        if (stringToConvert.equals("")) {
+            throw new IllegalArgumentException("Value is empty");
```

Comment:
Maybe you could combine this two exceptions, null and empty as invalid arguments.

Reply:
Thanks. I just have done that